spec/features/stations/admin_can_edit_a_station_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+feature 'On the stations path' do
+  context 'an Admin user' do
+    scenario 'can edit a station from station/edit' do
+      admin = create(:admin)
+      stations = create_list(:station, 35)
```

Comment:
I'm getting something weird stubbing this out where my erb file has a different set of stations than everywhere else and I'm not sure why